packages/manager/apps/telecom/src/app/telecom/pack/slots/task/index.js

```diff
@@ -0,0 +1,29 @@
+import angular from 'angular';
+import ngOvhTelecomUniverseComponents from '@ovh-ux/ng-ovh-telecom-universe-components';
+import '@uirouter/angularjs';
+import '@ovh-ux/ng-translate-async-loader';
+import 'angular-translate';
+import 'ovh-api-services';
+
+import controller from './task.controller';
+import template from './task.html';
+
+const moduleName = 'PackTask';
```

Comment:
```suggestion
const moduleName = 'ovhManagerTelecomPackSlotsTask';
```

As done in https://github.com/ovh/manager/blob/master/packages/manager/apps/telecom/src/app/telecom/telephony/telephony.module.js#L10